tests/grace_period_test.go

```diff
@@ -0,0 +1,229 @@
+package tests
+
+import (
+	"context"
+	"testing"
+
+	"cosmossdk.io/math"
+	transfertypes "github.com/cosmos/ibc-go/v6/modules/apps/transfer/types"
+	test "github.com/decentrio/rollup-e2e-testing"
+	"github.com/decentrio/rollup-e2e-testing/cosmos"
+	"github.com/decentrio/rollup-e2e-testing/ibc"
+	"github.com/decentrio/rollup-e2e-testing/relayer"
+	"github.com/decentrio/rollup-e2e-testing/testreporter"
+	"github.com/decentrio/rollup-e2e-testing/testutil"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+)
+
+// TestStart is a basic test to assert that spinning up a dymension network with 1 validator works properly.
+func TestGracePeriodCompliance(t *testing.T) {
```

Comment:
Wrong comment description. Can you update it?
Also `TestGracePeriodCompliance` function could be named `IBCTestGracePeriodCompliance`
File could be named `ibc_grace_period_test.go`

Reply:
Good catch! Tks

Reply:
Good catch. My typo